meta-psa/conf/layer.conf

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2019 Arm Limited and Contributors. All rights reserved.
+#
+# SPDX-License-Identifier: MIT
+#
+# psa layer configuration file
+BBPATH .= ":${LAYERDIR}"
+
+# We have a recipes directory, add to BBFILES
+BBFILES += "${LAYERDIR}/recipes*/*/*.bb ${LAYERDIR}/recipes*/*/*.bbappend"
+
+BBFILE_COLLECTIONS += "psa"
```

Comment:
I think we should use `meta-psa` or `psa-layer` as the `BBFILE_COLLECTIONS`

Reply:
Okay. I'll change it to "meta-psa" (to be consistent with "meta-mbl-distro").

Reply:
https://github.com/ARMmbed/meta-mbl/pull/576/commits/38945a37a480447ab9be489d67be4714838a138e